<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buckshot</class>
 <widget class="QDialog" name="buckshot">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SARSoft Buckshot</string>
  </property>
  <widget class="QLineEdit" name="coordsField">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Coordinates</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>871</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter only the numbers from the raw coordinates that you were given.
Do not use spaces or any punctuation.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="goButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>680</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Create Markers</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="markerNameField">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>630</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>620</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Marker Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="DdField">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="DMmField">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>421</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="DMSsField">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>430</y>
     <width>421</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Decimal Degrees
(Dd)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Degrees and Decimal Minutes
(DMm)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Degrees, Minutes, and Decimal Seconds
(DMSs)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="URLField">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>700</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>690</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>URL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>871</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Possibilities</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>730</y>
     <width>501</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Optional.  URL of a saved SARSoft map, hosted locally or on current network.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gpxFileNameField">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>630</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>620</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>GPX</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>660</y>
     <width>511</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Optional.  Full .gpx filename.  File is not written until 'Create Markers' is clicked.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gpxBrowseButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>630</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>coordsField</tabstop>
  <tabstop>markerNameField</tabstop>
  <tabstop>gpxFileNameField</tabstop>
  <tabstop>URLField</tabstop>
  <tabstop>goButton</tabstop>
  <tabstop>DMSsField</tabstop>
  <tabstop>DdField</tabstop>
  <tabstop>DMmField</tabstop>
  <tabstop>gpxBrowseButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>coordsField</sender>
   <signal>textChanged(QString)</signal>
   <receiver>buckshot</receiver>
   <slot>calcLatLon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>841</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>733</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>goButton</sender>
   <signal>clicked()</signal>
   <receiver>buckshot</receiver>
   <slot>createMarkers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>680</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>688</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gpxBrowseButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>buckshot</receiver>
   <slot>gpxSetFileName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>823</x>
     <y>641</y>
    </hint>
    <hint type="destinationlabel">
     <x>796</x>
     <y>605</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>markerNameField</sender>
   <signal>textChanged(QString)</signal>
   <receiver>buckshot</receiver>
   <slot>markerNameChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>640</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>606</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calcLatLon()</slot>
  <slot>createMarkers()</slot>
  <slot>gpxSetFileName()</slot>
  <slot>markerNameChanged()</slot>
 </slots>
</ui>
